clrdefinitions.cmake

```diff
@@ -18,6 +18,7 @@ if (CLR_CMAKE_TARGET_ARCH_AMD64)
   add_definitions(-D_TARGET_64BIT_)
   add_definitions(-DDBG_TARGET_64BIT)
   add_definitions(-DDBG_TARGET_AMD64)
+  add_definitions(-DFEATURE_EH_FUNCLETS)
```

Comment:
All other `FEATURE_XXX` flags are defined later in the file, with a condition. It would be nice to follow that style for this one too.

Reply:
Fixed